package.json

```diff
@@ -40,7 +39,7 @@
     "jsdom": "3.0.0",
     "jshint": "^2.6.3",
     "react": "^0.14.0",
-    "react-router": "^1.0.3",
+    "react-router": "^2.4.0",
```

Comment:
@remarkablemark that patch should make react-engine seamlessly work with v1.x and 2.x. can you issue a PR?

@JustinMGaiam can you paste the error logs (for the one you encountered before freezing react-router to 1.0.3)?